### PR TITLE
Add BC OrgBook and BC Geocoder Lookups to SHAS Intake Form

### PR DESCRIPTION
**Pull request by wilwong89**
# Description

Adjusted intake form to search OrgBook and Geocoder APIs for BC registered businesses and addresses.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have checked that unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: frontend/src/components/intake/ShasIntakeForm.vue [hunk 1]
```diff
@@ -189,6 +254,7 @@ onBeforeMount(async () => {
 </script>
 
 <template>
+  <div>{{ formRef?.values }}</div>
```
Comment (sanjaytkbabu) [inline frontend/src/components/intake/ShasIntakeForm.vue:257]: consider getting rid of this if dev is complete

**Comment by sanjaytkbabu** [comment]
Getting the following error when searching for an address:
"Error: Error
Cannot read properties of undefined (reading 'geocoderApi')"

File: app/src/routes/v1/submission.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ router.get(
 );
 
 // Submission create draft endpoint
-router.put('/draft', submissionValidator.createDraft, (req: Request, res: Response, next: NextFunction): void => {
```
Comment (sanjaytkbabu) [inline app/src/routes/v1/submission.ts:26]: Wondering if the validator isn't needed anymore?

Comment (wilwong89) [reply app/src/routes/v1/submission.ts:26]: I think this got accidentally added in one of the many merges for the `release/housing-intake` form. I recall this validator wasn't meant for this route.

**Pull request closed without merging**